### PR TITLE
 MODDICONV-35 - Changed type of jobProfile dataType field

### PR DESCRIPTION
**Pull request by RuslanLavrov**

### CODE REVIEW

File: schemas/mod-data-import-converter-storage/jobProfile.json [hunk 1]
```diff
@@ -17,11 +17,8 @@
     },
     "dataTypes": {
       "description": "Set of data types",
```
Comment (KaterynaSenchenko) [inline schemas/mod-data-import-converter-storage/jobProfile.json]: Please update the description and the property name to be singular

Comment (OleksandrYehorovEPAM) [inline schemas/mod-data-import-converter-storage/jobProfile.json]: I suppose description should be changed as well

File: examples/mod-data-import-converter-storage/jobProfileCollection.sample [hunk 2]
```diff
@@ -210,9 +194,7 @@
       "id": "b32e79bc-01d9-4d31-bc08-a3621fcfc1aa",
       "name": "Load EDI Invoice",
       "description": "",
```
Comment (Igor-Gorchakov) [inline examples/mod-data-import-converter-storage/jobProfileCollection.sample]: Why empty ?

Comment (KaterynaSenchenko) [reply examples/mod-data-import-converter-storage/jobProfileCollection.sample]: It's OK, description is an optional field, it can be empty

**Pull request merged**